.conform.yaml

```diff
@@ -0,0 +1,29 @@
+policies:
+  - type: commit
+    spec:
+      header:
+        length: 89
+        imperative: true
+        case: lower
+        invalidLastCharacters: .
+      body:
+        required: true
+      spellcheck:
+        locale: US
+      maximumOfOneCommit: false
+      conventional:
+        # Consistent with @commitlint/config-conventional, as referenced in Conventional Commits Specification.
+        # https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional#type-enum
+        types:
```

Comment:
question: why is `feat` and `fix` missing here? Is that intended or does it append these to some defaults?

Reply:
Yes, Feat and Fix are included by default. Not sure why those two and not any others.

Reply:
Ref: https://github.com/siderolabs/conform/blob/v0.1.0-alpha.27/internal/policy/commit/check_conventional_commit.go#L78

Reply:
Thanks for checking!